### ISSUE TITLE
Integrate ramp.network

### ISSUE THREAD

**Issue by manuelwedler** [raiden-network/raiden-wizard#317]
We would like to onboard users with fiat more easily. With ramp.network we could provide a modal which asks the user to buy ETH with fiat (Instead of sending with MetaMask). There should be a button placed next to the MetaMask button.

**Comment by Dominik1999** [comment]
On 30th of September we had a short discussion about legal topics regarding the integration of ramp.network into the Wizard with @jake-bb

We agreed on the two following outcomes:
1. We need to ad information to the Wizard's Terms and Conditions , see https://github.com/raiden-network/raiden-wizard/blob/bf2fe8662be4db2c36bcc920d8e31cec888a496e/resources/templates/index.html#L78-L90

2. We should check if the Wizard is still compliant to the data protection regulation - keeping in mind that the user is directly redirected to ramp.network if clicking on "Buy Eth"

@jake-bb will provide a suggestion until 10th of October.

**Comment by Dominik1999** [comment]
We implemented ramp.network into the Wizard and tested it. At the moment Wizard Users can only use the debit card option. Credit Cards, normal or instant bank transfers do not work or take too long.

We should decide whether the debit card coverage within our user group is big enough to proceed or not. If we proceed, we will continue with the legal analysis with Jim, see https://github.com/raiden-network/team/issues/856

We should have a decision until 23th of October

@GataKamsky @christianbrb
